--- lib2geom-1.1/2geom.pc.in.orig	2021-01-15 18:42:28.000000000 +0100
+++ lib2geom-1.1/2geom.pc.in	2022-02-08 16:21:55.187190307 +0100
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}/lib@LIB_SUFFIX@
 includedir=${prefix}/include
 
 Name: 2geom
--- lib2geom-1.1/CMakeLists.txt.orig	2021-01-15 18:42:28.000000000 +0100
+++ lib2geom-1.1/CMakeLists.txt	2022-02-08 17:17:50.041629876 +0100
@@ -147,4 +147,4 @@ install(DIRECTORY "${CMAKE_CURRENT_SOURC
 
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/2geom.pc.in
                 ${CMAKE_CURRENT_BINARY_DIR}/2geom.pc @ONLY IMMEDIATE )
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/2geom.pc" DESTINATION lib/pkgconfig)
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/2geom.pc" DESTINATION lib${LIB_SUFFIX}/pkgconfig)
